docs/4_preparing_roms_input_datasets.ipynb

```diff
@@ -146,8 +146,8 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    target_dir = os.path.join(os.getenv('PSCRATCH'), 'ROMS_TOOLS_INPUT_DATA')
</pre><p><code>target_dir = Path(os.getenv("PSCRATCH")) / "ROMS_TOOLS_INPUT_DATA"</code></p><p> </p><p>for consistency with C-Star docs and code elsewhere?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CWorthy-ocean/C-Star/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>